projects/packages/my-jetpack/_inc/context/notices/noticeTemplates.ts

```diff
@@ -2,6 +2,8 @@ import { __ } from '@wordpress/i18n';
 import { NOTICE_PRIORITY_HIGH } from '../constants';
 import { Notice } from './types';
 
+export const WELCOME_BANNER_NOTICE_IDS: string[] = [ 'site-connection-success-notice' ];
```

Comment:
I can't help but think there's a better name for this 🤔  But I can't think of one 😆